cmd/kflex/init/init.go

```diff
@@ -50,16 +50,6 @@ func Init(ctx context.Context, kubeconfig, version, buildDate string, domain, ex
 		util.PrintStatus("OpenShift cluster detected", done, &wg)
 	}
 	done <- true
-	util.PrintStatus("Installing shared backend DB...", done, &wg)
-	ensureSystemDB(ctx, isOCP)
-	done <- true
-
-	util.PrintStatus("Waiting for shared backend DB to become ready...", done, &wg)
-	util.WaitForStatefulSetReady(
-		*(client.GetClientSet(kubeconfig)),
-		util.GeneratePSReplicaSetName(util.DBReleaseName),
```

Comment:
Note that DBReleaseName needs to change from "postgres" to  "kubeflex-operator".